### DISCUSSION TITLE
scroll to index not aligning properly

### DISCUSSION THREAD

**Discussion by Itaysade98** [petyosi/react-virtuoso#747] [Q&A] [answered]
first of all I'd like to say the library is great and overall i've had a great experience with it.
i just noticed that when i use the scroll to index functionality the item i'm scrolling isn't aligned properly and it gets worse the lower i get in the list. this seems to be because my items have a margin - i don't know how the item location is calculated but the margin between items isn't taken into account i guess. if anyone has a solution i'll appreciate it!

**Comment by petyosi** [comment, marked as answer]
https://virtuoso.dev/troubleshooting#list-does-not-scroll-to-the-bottom--items-jump-around

**Reply by Itaysade98** [reply]
thanks for the quick answer! i don't know what i was thinking at the time, the easy solution for me was swapping the margin with empty spacer divs for every item with comfigured height since padding wasn't a good alternative for me.
